R/import.R

```diff
@@ -89,31 +89,49 @@
 }
 
 
-.import_news <- function(src_dir, tar_dir, tool) {
-  src <- .which_news(src_dir)
-  if (!is.null(src) && fs::file_exists(src)) {
-    tar <- fs::path_join(c(tar_dir, "NEWS.md"))
-    if (!fs::file_exists(tar)) {
-      cli::cli_alert_success("{.file NEWS} imported for the first time.")
-    }
-    fs::file_copy(src, tar_dir, overwrite = TRUE)
-    .parse_news(path = src_dir, news_path = tar)
+.import_news_changelog <- function(src_dir, tar_dir, name = "NEWS") {
+  src <- c(
+    "NEWS.md", "NEWS.txt", "NEWS", "NEWS.Rd",
+    "inst/NEWS.md", "inst/NEWS.txt", "inst/NEWS", "inst/NEWS.Rd"
+  )
+  src <- gsub("NEWS", name, src, fixed = TRUE)
+  src <- sapply(src, function(x) fs::path_join(c(src_dir, x)))
+  src <- Filter(fs::file_exists, src)
+
+  # no news to import
+  if (length(src) == 0) {
+    return(invisible())
+  # priority hard-coded by the order of the vector above
+  } else {
+    src <- src[1]
+  }
+
+  tar <- fs::path_join(c(tar_dir, paste0(name, ".md")))
+  first <- !fs::file_exists(tar)
+  if (first) {
+    cli::cli_alert_success("{.file {name}} imported for the first time.")
```

Comment:
I don't think the "for the first time" makes sense anymore, right?

Reply:
I mean, it "makes sense", because this won't appear when we just update the files for the second time. But I don't think it's particularly useful.

Reply:
yeah we can remove it